layouts/blog/post.html

```diff
@@ -20,6 +20,13 @@ <h1>{{ .Title }}</h1>
             </figure>
             {{ end }}
             {{ end }}
+            <a href="https://twitter.com/intent/tweet" class="twitter-share-button"
+                data-size="large"
+                data-text="{{ .Title }}"
+                data-url="{{ .Permalink }}"
+                data-hashtags="ApacheCamel"
+                data-show-count="false">Tweet this blog</a>
+            <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Unfortunately this won't work with our content security policy. I'm also concerned about tracking and adding weight to the page (5 requests, ~400KB). Perhaps we could do https://sharingbuttons.io/ instead?

Reply:
Thanks Zoran. I was thinking there might be such a case. sharingbuttons.io looks great. So, according to our policy:
https://camel.apache.org/privacy-policy/
we should no embed a js script that tracks users' activities, and as sharingbuttons.io claims that it only uses HTML & CSS and does not track users' activities we can use the widget in the blog template, right?